### PR TITLE
deploy/helm-chart: properly set dataset configuration

### PR DESCRIPTION
**Pull request by paulfantom**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._

This is allowing to configure promscale via configuration file and fixes issues with incorrectly setting dataset config values.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: deploy/helm-chart/values.yaml [hunk 1]
```diff
@@ -49,14 +49,15 @@ connection:
...
 
-# dataset configuration options. Values presented in this section are defaults. For full list of settings
-# and their default values go to https://github.com/timescale/promscale/blob/master/docs/dataset.md
-datasetConfig:
-  metrics:
-    compress_data: true
-    default_retention_period: 90d
-  traces:
-    default_retention_period: 30d
+# promscale configuration options. Values presented in this section are defaults. For full list of settings
```
Comment (cevian) [inline deploy/helm-chart/values.yaml:52]: Should we specify that this takes lower precedence than extraArgs?

**Review by cevian — APPROVED** [review]
LGTM. Want @antekresic to also review

File: deploy/helm-chart/values.yaml (hunk 1 above)
Comment (antekresic) [reply deploy/helm-chart/values.yaml:52]: I agree lets add a comment.

**Review by antekresic — APPROVED** [review]

File: deploy/helm-chart/values.yaml (hunk 1 above)
Comment (paulfantom) [reply deploy/helm-chart/values.yaml:52]: Good point :+1: I added a note about it.

**Pull request merged**